pyproject.toml

```
@@ -25,10 +25,10 @@ classifiers = [
     "Intended Audience :: Developers",
 ]
 dependencies = [
-    "typer >= 0.7.0",
-    "ghga-service-commons[api] >= 0.7.0",
-    "ghga-event-schemas >= 0.13.0",
-    "hexkit[akafka,s3,mongodb] >= 0.10.0",
+    "typer >= 0.9.0",
+    "ghga-service-commons[api] >= 1.1.0",
+    "ghga-event-schemas >= 1.0.0",
+    "hexkit[akafka,s3,mongodb] >= 0.11.1",
```

```suggestion
    "typer ~= 0.9.0",
    "ghga-service-commons[api] >= 1.1.0, <2",
    "ghga-event-schemas >= 1.0.0, <2",
    "hexkit[akafka,s3,mongodb] ~= 0.11.1",
```
This should now comply with draft SOP012 for a non-service library, right?